src/lib/const.ts

```diff
@@ -6,4 +6,4 @@ export const minSchemaVersion = 0;
 // maximal/current schema version the server supports
 export const maxSchemaVersion = 22;
 
-export const dnssdServiceType = "zwave-js-server";
+export const applicationName = "zwave-js-server";
```

Comment:
Let's create a new constant for this just so it's clear what we broadcast on DNS-SD